redash/models.py

```diff
@@ -28,10 +28,12 @@
 from sqlalchemy.event import listens_for
 from sqlalchemy.ext.mutable import Mutable
 from sqlalchemy.inspection import inspect
-from sqlalchemy.orm import backref, joinedload, object_session, subqueryload
+from sqlalchemy.orm import backref, joinedload, object_session
 from sqlalchemy.orm.exc import NoResultFound  # noqa: F401
 from sqlalchemy.types import TypeDecorator
 from functools import reduce
+from sqlalchemy_searchable import SearchQueryMixin, make_searchable, search
+from sqlalchemy_utils.types import TSVectorType
```

Comment:
Not that I mind much about adding `sqlalchemy_utils` as a dependency, but doesn't `sqlalchemy_searchable` provide something for this?

Reply:
Yeah, the [SQLAlchemy-Utils](https://github.com/kvesteri/sqlalchemy-utils) package is from the same author as [SQLAlchemy-Searchable](https://github.com/kvesteri/sqlalchemy-searchable) and contains a whole bunch of things.

The code behind [TSVectorType](https://github.com/kvesteri/sqlalchemy-utils/blob/master/sqlalchemy_utils/types/ts_vector.py) is trivial relatively speaking, so I guess we could also just ship our own. The only risk I see with this is that we could trade one dependency less for having to keep an eye on changes in SQLAlchemy-Searchable that require changes in the TSVectorType class as well.

On the other hand, although quite out of scope of this PR, is the question if there are things in SQLAlchemy-Utils that may be useful for Redash in general, given its swiss-army-knife-ness. I'm fine with whatever way you want to go with this.

Reply:
👍 ok, let's use it. No need to find more places to use it, we can gradually introduce it where applicable.